app/services/ovh-request/ovh-request.service.ts

```diff
@@ -8,26 +8,58 @@ let timestampValidity = 10000;
 
 @Injectable()
 export class OvhRequestService {
-  opts: any = {};
-  urlRoot: string = 'https://eu.api.ovh.com/1.0';
+  config: any;
   timestampServer: number;
   timestampLocal: number;
   timestampLag: number;
   timeRequestSended: boolean = false;
   timeObservable: Observable<any>;
 
   constructor(private http: Http) {
+    this.config = {
+      endpoint: '',
+      consumerKey: '',
+      validationUrl: '',
+      credentialToken: '',
+      urlRoot: '',
+      appKey: '',
+      appSecret: ''
+    };
+    for(var key in this.config) {
+      let value = localStorage.getItem(key);
+      if(value) {
+        this.config[key] = value;
```

Comment:
pq pas directement this.config[key] = localStorage.getItem(key); sans if et sans let value ?